.changeset/tough-mangos-repair.md

```diff
@@ -0,0 +1,8 @@
+---
+'@microsoft/parcel-transformer-markdown-html': patch
```

Comment:
I was testing locally, and upon running `npm run start` saw suggestion to run `npx update-browserslist-db@latest` which down the road made additional changes to package-lock file. Can you please commit them?

![image](https://github.com/microsoft/atlas-design/assets/57374379/536bb240-b94e-4c78-b35d-20790538755c)